modules/ztp-checking-kernel-rt-in-cluster.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-vdu-configuration-reference.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-checking-kernel-rt-in-cluster_{context}"]
+= Checking the realtime kernel version
+
+Always use the latest version of the realtime kernel in your {product-title} clusters. If you are unsure about the kernel version that is in use in the cluster, you can compare the current realtime kernel version to the release version with the following procedure.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
```

Comment:
Just noting the past/present perfect tenses from above here in case you decide to change it.